EventsExpress/Startup.cs

```diff
@@ -158,6 +161,7 @@ public void ConfigureServices(IServiceCollection services)
             services.AddMediatR(typeof(EventCreatedHandler).Assembly);
 
             services.AddAutoMapper(typeof(AutoMapperProfile).GetTypeInfo().Assembly);
+            services.AddAutoMapper(typeof(OccurenceEventAutoMapper).GetTypeInfo().Assembly);
```

Comment:
You don't have to do this line. The profile will be added automatically. Remove it please.

Reply:
Solved